skyportal/enum_types.py

```diff
@@ -1,16 +1,44 @@
-import sqlalchemy as sa
-from enum import Enum
+import astropy.units as u
 import inspect
+import numpy as np
+import sncosmo
+from sncosmo.bandpasses import _BANDPASSES
+from sncosmo.magsystems import _MAGSYSTEMS
+import sqlalchemy as sa
 
+from enum import Enum
 from baselayer.app.env import load_env
-
+from baselayer.log import make_log
 from . import facility_apis
 
-from sncosmo.bandpasses import _BANDPASSES
-from sncosmo.magsystems import _MAGSYSTEMS
+log = make_log('enum_types')
 
 _, cfg = load_env()
 
+# load additional bandpasses into the SN comso registry
+existing_bandpasses_names = [val['name'] for val in _BANDPASSES.get_loaders_metadata()]
+additional_bandpasses_names = []
+for additional_bandpasses in cfg.get('additional_bandpasses', []):
+    name = additional_bandpasses.get("name")
+    if not name:
+        continue
+    if name in existing_bandpasses_names:
+        log(
+            f"Additional Bandpass name={name} is already in the sncosmo registry. Skipping."
+        )
+    try:
+        wavelength = np.array(additional_bandpasses.get("wavelength"))
+        transmission = np.array(additional_bandpasses.get("transmission"))
+        band = sncosmo.Bandpass(wavelength, transmission, name=name, wave_unit=u.AA)
+    except Exception as e:
+        log(f"Could not make bandpass for {name}")
```

Comment:
```suggestion
        log(f"Could not make bandpass for {name}: {e}")
```

Not crucial, but will make grepping the logs easier.  Still somewhat regret not using structured logs!